docs/_snippets/csf-factories-main-example.md

```diff
@@ -0,0 +1,10 @@
+```ts filename=".storybook/main.ts" renderer="react" language="ts"
```

Comment:
When you can, add a `js` variant to ensure full snippet coverage and apply it to the other examples as well

Reply:
I adjusted the filename to use the `js|ts` construction.

I also confirmed that this snippet displays correctly, regardless of chosen language.

The snippets on this page don't need to accommodate older patterns. They can function as all snippets will when CSF 4 is stable.